aten/src/ATen/native/cuda/EmbeddingBag.cu

```diff
@@ -450,16 +447,16 @@ Tensor _embedding_bag_dense_backward_cuda(const Tensor &grad_, const Tensor &ind
   checkSameGPU("embedding_bag_cuda", grad_arg, indices_arg);
 
 
-  switch (mode) {
-    case MODE_SUM:
-    case MODE_MEAN:
-      if (mode == MODE_MEAN)
+  switch (static_cast<EmbeddingBagMode>(mode)) {
```

Comment:
if you want to go for the full refactor, you should probably never have static_casts, and always pass the right type around.

Reply:
I wasn't really planning on going all the way, at least in this PR. This refactor was just to centralize the mode enum (we were declaring constant ints in 2 places already).